### ISSUE TITLE
About the paper purpose

### ISSUE THREAD

**Issue by JinyiHan99** [akkeshav/confidence_probability_alignment#1]
Hello, thx for your work! I have read your original paper carefully, but i have some questions to ask you. We know that there are many works to explore the **inner confidence** of LLMs using the token logits, and there also many works to guide LLMs to express their  **verbalized certainty**.  Both types of work are aimed at improving the calibration ability of the LLM, which is to evaluate the quality of answers. Therefore, 
(1)**What is the purpose of exploring the correlation between these two metrics?** In my opinion, these two metrics are different ways to detect how sure the model is that the generated answer is the correct answer.
(2) **If the two shows strongest alignment, what does it mean? What can we do with this signal?**
 
 Look forward to your reply deeply! Thx again!!
